packages/core/src/v2/entity.ts

```diff
@@ -275,9 +278,9 @@ export interface EntityIdentifiable<T extends Entity> {
  */
 export function isSelectedProperty<EntityT extends Entity>(
   json,
-  selectable: Selectable<EntityT>
+  field: Field<EntityT> | Link<EntityT>
```

Comment:
see comments above on the selectable type.

Reply:
Same here. In this case we check whether a field is being deserialized (so only fields or links) is also selected.